### DISCUSSION TITLE
Simple Worker Status

### DISCUSSION THREAD

**Discussion by azriel46d** [Netflix/conductor#3084] [Ideas]
Currently, the UI there is no way of knowing the status of a worker. So the idea here is to be able to identify the status of a workers.

The suggestion would be to show
1. How many workers are associated with a task definition.
2. Whats the status , determined on the last queue check, the idea would be to check whether a worker is alive (green), degraded (yellow), or offline (red).

**Comment by aravindanr** [comment]
This would require each worker to register with conductor and report heartbeats. Decoupled workers is a deliberate decision in Conductor, since we don't want it to manage workers. There are other means of identifying worker health, like exposing a health endpoint.

**Reply by aravindanr** [reply]
The PollData tab in the UI shows what Conductor knows about a worker.
